includes/razorpay-webhook.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
```

Comment:
We want to give a absolute URL to the user that they can enter in the Razorpay dashboard in the settings page.